src/main/java/uk/gov/gchq/gaffer/doc/function/CurrentTimeExample.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import uk.gov.gchq.koryphe.impl.function.CurrentTime;
+
+public class CurrentTimeExample extends FunctionExample {
+
+    public static void main(final String[] args) {
+        new CurrentTimeExample().runAndPrint();
+    }
+
+    public CurrentTimeExample() {
+        super(uk.gov.gchq.koryphe.impl.function.CurrentTime.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        getCurrentTime();
+    }
+
+    private void getCurrentTime() {
+        // ---------------------------------------------------------
+        final CurrentTime currentTime = new CurrentTime();
+        // ---------------------------------------------------------
+
+        runExample(currentTime, null);
```

Comment:
This doesn't look right. You still need to give it some test inputs otherwise an empty table gets rendered on the screen:

Input Type | Input | Result Type | Result
-- | -- | -- | --

Reply:
These have empty input.

CancelScheduledJobExample.java
CountExample.java
CountGroupsExample.java
CurrentTimeExample.java
ExportToGafferResultCacheExample.java
ExportToSetExample.java
ForEachExample.java
GenerateElementsExample.java
GenerateObjectsExample.java
GetAdjacentIdsExample.java
GetAllElementsExample.java
GetAllJobDetailsExample.java
GetDataFrameOfElementsExample.java
GetElementsBetweenSetsExample.java
GetElementsExample.java
GetElementsInRangesExample.java
GetElementsWithinSetExample.java
GetGafferResultCacheExportExample.java
GetGraphFrameOfElementsExample.java
GetJavaRDDOfAllElementsExample.java
GetJavaRDDOfElementsExample.java
GetJobDetailsExample.java
GetJobResultsExample.java
GetSetExportExample.java
GetTraitsExample.java
JoinExample.java
LimitExample.java
MaxExample.java
MinExample.java
NamedOperationExample.java
ReduceExample.java
SortExample.java
ToArrayExample.java
ToCsvExample.java
ToEntitySeedsExample.java
ToListExample.java
ToMapExample.java
ToSetExample.java
ToSingletonListExample.java
ToStreamExample.java
ToVerticesExample.java